TUM Campus App/CardKey.swift

```diff
@@ -30,9 +34,17 @@ enum CardKey: Int, Codable {
             return "Tuition"
         case .bookRental:
             return "Book Rental"
+        case .mvg:
+            return "MVG"
+        case .grades:
+            return "Grades"
+        case .studyRooms:
+            return "Study Rooms"
+        case .lectures:
+            return "Lectures"
         }
     }
     
-    static var all: [CardKey] = [.tufilm, .calendar, .news, .cafeteria, .tuition, .bookRental,]
+    static var all: [CardKey] = [.tufilm, .calendar, .news, .cafeteria, .tuition, .bookRental, .mvg, .grades, .studyRooms, .lectures,]
```

Comment:
We might have to add these to `PersistentCardOrder` manually after the updates. Otherwise the users won't see them ;)

Reply:
See #212